CONTRIBUTING.adoc

```diff
@@ -550,6 +551,32 @@ vector: 'logos/${imageId}/${imageId}.svg'
 credit: 'Your Name'
 credit_url: 'https://twitter.com/yourtwitteraccount'
 ```
+=== Adding a book
+
+When creating a submission for the link:https://www.jenkins.io/books/[Jenkins Books] page, prepare a `.yml` file with the following configuration:
```

Comment:
```suggestion
When creating a submission for the link:/books/[Jenkins Books] page, prepare a `.yml` file with the following configuration:
```

Reply:
I'm afraid that's not correct and doesn't work. Kevin's original commit was right.

Reply:
Let me revert this.

Reply:
Actually this was done in a few other places on this page too. I have just pushed a commit to fix these as wel.